eslint-custom-rules/no-raw-text.js

```diff
@@ -0,0 +1,35 @@
+const ALLOWED_JSX_TAGS = [
+  /^.*Text$/,
+  /^Styled+(Hero|Title[1-4]|Body|ButtonText(|NeutralInfo|Primary|Secondary)|Caption(|NeutralInfo|Primary|Secondary))$/,
+]
+
+const WHITESPACES_REGEX = /^\s+$/
```

Comment:
est-ce qu'on veut autoriser les textes vides ?

```suggestion
const WHITESPACES_REGEX = /^\s*$/
```

Reply:
@voisinhugo tu penses que ça rentre dans le scope de ce qu'on voulait ignorer ?

Reply:
Je connais qu'un cas de textes vides, c'est quand on fait :
```
<Toto>{toto}</Toto>
```
ou
```
<Toto><Tata/></Toto>
```
(autofix par notre linter)

Donc je pense qu'on veut autoriser le 1er cas, donc autoriser les textes vides 👍